docs/guides/slash_commands.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: Slash commands
+author: l7ssha
+timestamp: 2021-09-21
+category: guides
+sidebar_position: 7
+---
+

```

Comment:
Maybe add a warning that using the raw nyxx_interactions API is more complicated than using nyxx_commands for commands, and should only be used if developers are curious or have specific needs nyxx_commands can't fulfil?

Reply:
Hm, I would opt for an information rather than a warning 🤔
It's up to the end-user to choose whether or not use this package instead of `nyxx_commands`.
Actually, if they want to shoot themself in the foot, it's their choice.

Reply:
An information would also work. The idea is to not confuse people as this guide is titled `Slash Commands` but is really specific to nyxx_intetractions.